primitives/arithmetic/src/fixed_point.rs

```diff
@@ -797,8 +809,8 @@ macro_rules! implement_fixed {
 				let a = $name::saturating_from_integer(42);
 				assert_eq!(a.into_inner(), 42 * accuracy);
 
-				let a = $name::saturating_from_integer(-42);
-				assert_eq!(a.into_inner(), -42 * accuracy);
+				// let a = $name::saturating_from_integer(-42);
```

Comment:
leftover.

Reply:
To me it seems like the assertion is correct and should still hold
```suggestion
				let a = $name::saturating_from_integer(-42);
```